assets/scss/_variables.scss

```diff
@@ -1,2 +1,21 @@
+// Colors
 $font-light: #fff;
+$main-color: #0D4C5E;
 $secondary-color: #b4437b;
+
+// Font families
+$dancing-script: 'Dancing Script', cursive, sans-serif;
+$playfair: 'Playfair Display', sans-serif;
+$osfald: "Oswald", sans-serif;
+$montserrat: "Montserrat", sans-serif;
+$roboto: "Roboto", sans-serif;
+$roboto-condensed: "Roboto Condensed", sans-serif;
```

Comment:
my bad, on the design was `"Roboto Condensed"` however it would be too many fonts so you can remove it and use Roboto instead